db/migrate/20170405065305_remove_image_password_audit.rb

```diff
@@ -0,0 +1,16 @@
+class RemoveImagePasswordAudit < ActiveRecord::Migration
+  def up
+    Audited::Audit.where(:auditable_type => 'Image').where('audited_changes like ?', '%password%').find_each do |audit|
+      audit.audited_changes.delete("password")
+      if audit.audited_changes.empty?
+        audit.destroy
```

Comment:
i think we should show an audit entry when the password was changed, just dont show the password itself.